irohad/ametsuchi/impl/redis_block_query.hpp

```diff
@@ -36,7 +36,7 @@ namespace iroha {
       RedisBlockQuery(cpp_redis::redis_client &client, FlatFile &file_store);
 
       rxcpp::observable<model::Transaction> getAccountTransactions(
-          const std::string &account_id) override;
+          const std::string &account_id, const model::Pager &pager) override;
```

Comment:
Is it possible to make default parameter for pager?

Reply:
Only via global var (with this fn signature), that's not really good

Reply:
Yes, if using default parameter, global variable is needed
```cpp
auto NO_PAGER = iroha::model::Pager{EMTPYHASH, MAX_LIMIT};
...
or
```cpp
auto NO_PAGER = iroha::model::Pager{EMPTY_HASH, -1};
```
which are not good, I think.

Reply:
This is not was I expect. I tried to provide an example of default behavior:
```
#include <iostream>
#include <string>

void foo(const std::string& val = std::string("1")) {
    std::cout << val << std::endl;
}

int main()
{
  foo("bar");
  foo();
}
```
Can we use same approach for `Pager` parameter here?

Reply:
So should I write something like this?
```cpp
rxcpp::observable<model::Transaction> getAccountTransactions(
    const std::string &account_id, const model::Pager &pager = model::Pager{"", 99999999})
```

Reply:
Yes, but with a maximal number on the page.

Reply:
Add default parameter to BlockQuery interface.